dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/hash/Murmur1HashPartitioner.java

```diff
@@ -40,6 +40,14 @@ public Long hash(String key) {
         byte[] b = bb.array();
         return UnsignedIntsUtils.toLong(Murmur1Hash.hash(b, b.length));
```

Comment:
call the byte array implementation here

Reply:
the byte[] implementation does not have the length.